iothub/device/tests/Authentication/IotHubConnectionStringBuilderTests.cs

```diff
@@ -44,6 +108,22 @@ public void IotHubConnectionStringBuilder_ParamConnectionString_ParsesDeviceId()
             iotHubConnectionCredentials.DeviceId.Should().Be(DeviceId);
         }
 
+        [TestMethod]
+        public void IotHubConnectionStringBuilder_ParamConnectionString_ParsesTokenRenewalBuffer()
+        {
+            var connectionString = $"HostName={HostName};SharedAccessKeyName={SharedAccessKeyName};DeviceId={DeviceId};SharedAccessKey={SharedAccessKey}";
+            var iotHubConnectionCredentials = new IotHubConnectionCredentials(connectionString);
+            iotHubConnectionCredentials.SasTokenRenewalBuffer.Should().Be(0);
```

Comment:
wouldn't this be resolved to the default buffer?

Reply:
elaborate?